ext/server/command.cpp

```diff
@@ -222,9 +222,21 @@ void extract_scalar<Tango::DEV_ENCODED>(const CORBA::Any &any, boost::python::ob
         throw_bad_type(Tango::CmdArgTypeName[Tango::DEV_ENCODED]);
     
     bopy::str encoded_format(data[0].encoded_format);
+
+#if PY_MAJOR_VERSION >= 3
+    boost::python::object encoded_data(
```

Comment:
couldn't we use the new code for both python 2 and python 3?

Reply:
I  guess so, it's in c131db3.